app/api/__init__.py

```diff
@@ -89,7 +89,7 @@
 # users
 api.route(UserList, 'user_list', '/users', '/events/<int:event_id>/organizers')
 api.route(UserDetail, 'user_detail', '/users/<int:id>', '/notifications/<int:notification_id>/user',
-          '/event-invoices/<int:event_invoice_id>/user', '/speakers/<int:speaker_id>/user',
+          '/event-invoices/<event_invoice_identifier>/user', '/speakers/<int:speaker_id>/user',
```

Comment:
Instead of replacing it with an identifier, I would suggest making a separate endpoint, and let the `id` functionality be there as well. @iamareebjamal said that android uses `event_id` at most places to retrieve resources. And at frontend, we are using the identifier in most places.
@iamareebjamal @CosmicCoder96 Your views on this?

Reply:
I'm not changing `event_id` anywhere. The thing I'm changing is `event_invoice_id` to `event_invoice_identifier` and as far as I know we haven't implemented event invoice feature on any of the clients till now.

Reply:
As we know that all the order endpoints are implemented using `order_identifier`, I'm trying to make event_invoice similar in functionality as both are similar in many features.

Reply:
Support both